release_creation/bundle/requirements/v0.0.latest.requirements.txt

```diff
@@ -12,8 +11,9 @@ dbt-databricks @ git+https://github.com/databricks/dbt-databricks.git@1.8.latest
 dbt-trino @ git+https://github.com/starburstdata/dbt-trino.git@master
 dbt-fabric @ git+https://github.com/microsoft/dbt-fabric.git@main
 dbt-synapse @ git+https://github.com/microsoft/dbt-synapse.git@master
-grpcio-status~=1.47.0
+
 pyasn1-modules~=0.2.1
-pyarrow!=12.0.1
+pyarrow~=14.0.1
```

Comment:
This was also an old holdover. It looks like this change was made in the minor reqs files in `1.4`.